css/styles.css

```diff
@@ -59,12 +70,29 @@
   border: solid;
 }
 
-.control-me::after {
+#answer-checkbox {
```

Comment:
please use a class here, the hack will still work

Reply:
and everywhere else where you used this id for styling

Reply:
will try and see if it works